ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -471,6 +473,34 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Transmission',
+      type: 'GainsEffect',
+      // CF3 Chelomorph (Wing icon - cleave behind player)
+      // D48 Glossomorph (Snake icon - cleave in front of player)
+      netRegex: NetRegexes.gainsEffect({ effectId: ['CF3', 'D48'] }),
+      condition: Conditions.targetIsYou(),
+      preRun: (data, matches) => {
+        // 1st transmission has 11s duration, 2nd has 25s duration
+        // in either case, trigger should fire 3s before debuff expires
+        const duration = parseFloat(matches.duration);
+        data.transmissionAlertDelay = duration > 3 ? duration - 3 : 0;
+      },
+      delaySeconds: (data) => {
+        return data.transmissionAlertDelay;
+      },
+      infoText: (_data, matches, output) => {
+        return matches.effectId === 'D48' ? output.forwardCleave!() : output.backwardCleave!();
+      },
+      outputStrings: {
+        forwardCleave: {
+          en: 'Cleave in Front (Look Away)',
+        },
+        backwardCleave: {
+          en: 'Cleave Behind (Face Boss)',
```

Comment:
Just because English is quirky and "look" and "face" can mean the same thing here but wildly different things in other contexts, can we standardize these two parentheses to have more similar verbiage, ie. "Look Out" / "Look In" or similar? Or does it feel like there's value in having them be different enough that they don't bleed together mentally during raid?

Reply:
Another suggestion is "Front Cleave" and "Rear Cleave" to keep it short.

Reply:
Thanks - didn't think about the linguistic nuance.  I struggled a bit with how prescriptive to be on where to look, since, e.g., some groups have all players face the boss (at least during trans1) and just put snakes in front/wings behind, so everyone can face the boss..  I think "Front Cleave" and "Rear Cleave" is a good solution - and if people want a specific instruction about where to aim it, they can always modify it in config. :-)